__tests__/adapters/vfs/system.js

```diff
@@ -95,6 +80,40 @@ describe('VFS System adapter', () => {
       .toBe(true);
   });
 
+  test('#stat - file', () => {
+    const realPath = path.join(core.configuration.tempPath, 'jest/test');
+
+    return expect(request('stat', 'home:/test', createOptions()))
+      .resolves
+      .toMatchObject({
+        filename: 'test',
+        path: realPath,
+        size: 0,
+        isFile: true,
+        isDirectory: false,
+        mime: 'application/octet-stream',
+        totalCount:null,
```

Comment:
Can you make sure you have spaces between your properties and their values? Thanks :)

Reply:
Sorry for that, I fixed it.